.github/workflows/release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     outputs:
       release_url: ${{ steps.create_release.outputs.upload_url }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
I can't comment on the actual line, but there's still a `set-output` left on line 78

That should probably become `echo "name=asset_file::${REL_TAG}.tar.gz" >> $GITHUB_OUTPUT`